diff -Naru python-dmidecode-3.12.3.org/src/dmidecode.c python-dmidecode-3.12.3.sw/src/dmidecode.c
--- python-dmidecode-3.12.3.org/src/dmidecode.c	2022-11-17 13:51:28.000000000 +0800
+++ python-dmidecode-3.12.3.sw/src/dmidecode.c	2022-12-06 11:13:37.911806178 +0800
@@ -2,10 +2,10 @@
 /*. ******* coding:utf-8 AUTOHEADER START v1.1 *******
  *. vim: fileencoding=utf-8 syntax=c sw=2 ts=2 et
  *.
- *. © 2007-2013 Nima Talebi <nima@autonomy.net.au>
- *. © 2009-2013 David Sommerseth <davids@redhat.com>
- *. © 2002-2010 Jean Delvare <khali@linux-fr.org>
- *. © 2000-2002 Alan Cox <alan@redhat.com>
+ *. Â© 2007-2013 Nima Talebi <nima@autonomy.net.au>
+ *. Â© 2009-2013 David Sommerseth <davids@redhat.com>
+ *. Â© 2002-2010 Jean Delvare <khali@linux-fr.org>
+ *. Â© 2000-2002 Alan Cox <alan@redhat.com>
  *.
  *. This file is part of Python DMI-Decode.
  *.
@@ -882,6 +882,7 @@
           { 0x2F, "Core m7" },
 
           { 0x30, "Alpha" },
+	   { 0x9916, "Sw_64" },
           { 0x31, "Alpha 21064" },
           { 0x32, "Alpha 21066" },
           { 0x33, "Alpha 21164" },
